### DISCUSSION TITLE
Getting error on startup after migrating to 0.6.2

### DISCUSSION THREAD

**Discussion by anazar2001** [abiosoft/colima#872] [Q&A]
Getting this error now - [hostagent] Waiting for the essential requirement 3 of 5: "sshfs binary to be installed". It looks like it gets stuck at this step. Any ideas? Never had issues like this with 0.5.2

**Comment by rfay** [comment]
You know that you need to `colima delete` (losing all volumes, containers, images) and then `colima start` on upgrade to 0.6.2 from 0.5.2, right?

**Reply by anazar2001** [reply]
did that.

brew update -> brew upgrade colima -> colima delete -> colima start

colima version shows 0.6.2

**Reply by rfay** [reply]
I guess it's 
* https://github.com/abiosoft/colima/issues/871
